user.js

```diff
@@ -202,7 +202,7 @@ user_pref("mail.rights.override", true);  // [DEFAULT: unset]
  * $url Web page is opened if $viewed is lower than $version (and 0330 policy bypass notification is disabled)
  * [1] https://searchfox.org/comm-esr115/source/mail/base/content/messenger.js#455 ***/
    // user_pref("app.donation.eoy.version", 2);
-user_pref("app.donation.eoy.version.viewed", 999);
+// user_pref("app.donation.eoy.version.viewed", 999);
```

Comment:
```suggestion
    // user_pref("app.donation.eoy.version.viewed", 999);
```

The rationale here was : users that come across this template must be already aware of FOSS ecosystem constraints. On the same idea, before looking for hardened configuration like this, you must have started TB at least once, so you would have already encountered this notification.
Moreover, when we deal with multiple profiles and create some "regularly", all of these pop-ups and banners are very annoying.
No strong opinion on the "start page" above, but I'd keep this one disabled.

Reply:
they may be annoying but have no security relevance and indirectly harm the project, making it less maintained and thus less secure.

I would keep it

Reply:
Indeed, and I hate "authoritative maintaining" decisions, let's ask others what they think about it :

Hello to both of you @atomGit & @Dupond :slightly_smiling_face: 
Could you please read the arguments above and tell us what you think about the default value we should keep for "donations appeals" banner preference ? Thanks :pray: